src/utils/platform.ts

```diff
@@ -1,56 +1,38 @@
-export function isChromiumBased() {
-    return navigator.userAgent.toLowerCase().includes('chrome') || navigator.userAgent.toLowerCase().includes('chromium');
-}
-
-export function isFirefox() {
-    return navigator.userAgent.includes('Firefox');
-}
-
-export function isVivaldi() {
-    return navigator.userAgent.toLowerCase().includes('vivaldi');
-}
-
-export function isYaBrowser() {
-    return navigator.userAgent.toLowerCase().includes('yabrowser');
-}
-
-export function isOpera() {
-    const agent = navigator.userAgent.toLowerCase();
-    return agent.includes('opr') || agent.includes('opera');
-}
-
-export function isEdge() {
-    return navigator.userAgent.includes('Edg');
-}
-
-export function isWindows() {
-    if (typeof navigator === 'undefined') {
-        return null;
-    }
-    return navigator.platform.toLowerCase().startsWith('win');
-}
-
-export function isMacOS() {
-    if (typeof navigator === 'undefined') {
-        return null;
+import {PlatformData} from '../definitions';
+
+export const platformData: PlatformData = {} as any;
+
+export function runPlatformTest() {
```

Comment:
Is there a need in this function? You could just assign everything in the top level scope of this file.

```typescript
export const isChromium = userAgent.includes('chrome') || userAgent.includes('chromium');
export const isFirefox = userAgent.includes('firefox');
```

If the tests will fail (they should not in JSDom environment), substitute `navigator` with some object.